package.json

```diff
@@ -14,28 +14,34 @@
   },
   "types": "dist/index.d.ts",
   "scripts": {
-    "prebuild": "rm -rf dist && yarn run codegen",
+    "prebuild": "rm -rf dist && pnpm run codegen",
```

Comment:
Why replace yarn with pnpm?

Reply:
Not a problem, just wanted to understand the reasoning.

cc @anxolin and @shoom3301 for visibility